src/include/pbs_ifl.h

```diff
@@ -406,7 +411,7 @@ enum mgr_obj {
 #ifndef MAXNAMLEN
 #define MAXNAMLEN		255
 #endif
-
+#define PBS_MAXSCHEDNAME 	15
```

Comment:
This is in line with maximum value for PBS queue name i.e. PBS_MAXQUEUENAME.  Just to maintain the consistency we used the same length for scheduler as well. We can increase the length if required.